lib/fuse_versionscript

```diff
@@ -129,6 +129,7 @@ FUSE_3.0 {
 		fuse_parse_conn_info_opts;
 		fuse_fs_lseek;
 		fuse_reply_lseek;
+		fuse_get_conn_info;
```

Comment:
That is not right here, but needs a new `FUSE_3.17 {  } FUSE_3.12;` section, as that symbol is not part of libfuse-3.0.

 I'm going to push a patch today to increase master to 3.17.

Reply:
> That is not right here, but needs a new `FUSE_3.17 { } FUSE_3.12;` section, as that symbol is not part of libfuse-3.0.
> 
> I'm going to push a patch today to increase master to 3.17.

Hi, have you finished this patch? I need to rebase my patch on top of it, thanks.